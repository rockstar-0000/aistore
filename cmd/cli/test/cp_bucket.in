ais archive gen-shards "ais://$BUCKET_1/prune/shard-{001..500}.tar" // IGNORE

ais cp ais://$BUCKET_1 ais://$BUCKET_2 --sync --wait // IGNORE
ais ls ais://$BUCKET_2 --prefix prune | grep Listed

ais rmo "ais://$BUCKET_1/prune/shard-{251..260}.tar" --wait // IGNORE
ais ls ais://$BUCKET_1 --prefix prune | grep Listed

ais cp ais://$BUCKET_1 ais://$BUCKET_2 --sync --wait // IGNORE
ais ls ais://$BUCKET_2 --prefix prune | grep Listed

ais rmo "ais://$BUCKET_1/prune/shard-{351..360}.tar" --wait // IGNORE
ais ls ais://$BUCKET_1 --prefix prune | grep Listed

ais cp "ais://$BUCKET_1/prune/shard-{001..500}.tar" ais://$BUCKET_2 --sync --wait // IGNORE
ais ls ais://$BUCKET_2 --prefix prune | grep Listed
