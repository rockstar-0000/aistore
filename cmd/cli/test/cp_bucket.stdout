Listed 500 names
Listed 490 names
Listed 490 names
Listed 480 names
Listed 480 names
